tensorflow/lite/kernels/reverse_test.cc

```diff
@@ -12,12 +12,12 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 ==============================================================================*/
+#include <gmock/gmock.h>
+#include <gtest/gtest.h>
```

Comment:
These lines should go back to where they were.

Reply:
Addressed the requested change in [#1b42f81](https://github.com/MCW-Dev/tensorflow/pull/31/commits/1b42f812cab737e1f4afff6d89506aa5c1aeb0bc)